prodtests/full-system-test/run-workflow-on-inputlist.sh

```diff
@@ -17,10 +17,15 @@ if [[ $2 != "LOCAL" && ! -f $2 ]]; then
   exit 1
 fi
 
+if [[ $1 == "CTF" ]]; then
```

Comment:
I don't understand this setting. EXTINPUT should not be set from the outside anyway. So this should not be needed.

Reply:
Actually, Now I remember I think we discussed this with Ole before: At the beginning of this script, we should actually check for multiple variables, which are set by this script, that they are not set from the outside. I.e.
```
for i in EXTINPUT CTFINPUT RAWTFINPUT; do
  [[ ! -z ${!i} ]] && { echo "$i must not be set!"; exit 1; }
done
```

Reply:
I think this is a leftover from when we were still using the EPN mode, so it can be safely removed. The check at the beginning would be helpful